### PR TITLE
Feat placeholder #1748

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [ ] Bug fix 
- [X] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [X] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [X] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
  - [X] I have updated the documentation accordingly.
- [X] My change requires a change to [Typescript typings](./types/lib).
  - [X] I have updated the typings accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by illiteratewriter** [comment]
I've also updated the Bootstrap version

File: src/Placeholder.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { mapToCssModules, tagPropType } from './utils';
+import { getColumnClasses } from './Col';
+
+const propTypes = {
```
Comment (phwebi) [inline src/Placeholder.js:7]: What do you think about just extending `Col.propTypes`? Seems like `Col` props are the main method of specifying `Placeholder` widths.
```
const propTypes = {
  ...Col.propTypes,
  color: PropTypes.string,
  tag: tagPropType,
  animation: PropTypes.oneOf(['glow', 'wave']),
  innerRef: PropTypes.oneOfType([PropTypes.object, PropTypes.func, PropTypes.string]),
  size: PropTypes.oneOf(['lg', 'sm', 'xs']),
};
```

File: src/__tests__/Placeholder.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  it('should render color', () => {
+    const wrapper = shallow(<Placeholder xs={12} color="primary" />);
+    expect(wrapper.hasClass('bg-primary')).toBe(true);
+  })
+
+  it('should render size', () => {
+    const wrapper = shallow(<Placeholder size="lg" xs={12} />);
+    expect(wrapper.hasClass('placeholder-lg')).toBe(true);
+  })
+})
```
Comment (phwebi) [inline src/__tests__/Placeholder.spec.js:30]: Can we add tests for:

1. Specifying col widths for different size breakpoints?
```
<Placeholder size="lg" xs={12} lg={7} />
```

2. Verifying usage of the `widths` prop?

**Review by phwebi — COMMENTED** [review]
Thanks so much for helping with the Bootstrap 5 upgrade!

**Review by phwebi — APPROVED** [review]
Looks great! Thank you 😄

**Pull request merged**